components/eamxx/src/physics/p3/CMakeLists.txt

```diff
@@ -113,10 +111,6 @@ foreach (file IN ITEMS ${P3_TABLES})
   GetInputFile(${file})
 endforeach()
 
```

Comment:
I can't add a comment out of the lines you modified, so I'm writing here.

Since we have NO fortran left, we should remove the lines

```cmake
  set_target_properties(${P3_LIB} PROPERTIES
    Fortran_MODULE_DIRECTORY ${P3_LIB}/modules
  )
```
as well as the line `    ${CMAKE_CURRENT_BINARY_DIR}/${P3_LIB}/modules` inside the `target_include_directories` call (we don't generate fortran .mod files anymore)